lcm-cmake/lcmUtilities.cmake

```diff
@@ -84,6 +84,7 @@ function(_lcm_add_python_package PATH)
     if(EXISTS ${_init})
       file(REMOVE ${_init})
     endif()
+    file(APPEND ${_init} "\"\"\"LCM package __init__.py file\nThis file automatically generated by lcmUtilities.cmake.\nDO NOT MODIFY BY HAND!!!!\n\"\"\"\n\n")
```

Comment:
Would `lcm_wrap_types` be more appropriate than `lcmUtilities.cmake`, I wonder? Also, can you please split this to keep the line length under 80?